Google.PowerShell/ReleaseFiles/GoogleCloud.psm1

```diff
@@ -51,51 +54,95 @@ function Install-GCloudSdk {
              " set `$env:GCLOUD_SDK_INSTALLATION_NO_PROMPT to true."
     $caption = "Installing Google Cloud SDK"
 
+    $uiQuery = "Do you want to install non-interactively? Select no to install through the interactive installer."
```

Comment:
This double-negative can be hard to reason. Ask "Do you want to use the interactive installer?" and invert the ShouldContinue if statement.